great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -546,7 +546,7 @@ def _build_numeric_columns_rule() -> Rule:
             ],
             "allow_relative_error": False,
             "false_positive_rate": 0.05,
-            "estimator": "bootstrap",
+            "estimator": "exact",
```

Comment:
@NathanFarmer I would prefer that we leave `bootstrap` as is.  `DataAssistantRunner.run()` already sets `exact` by default.

Reply:
@alexsherstinsky we should use `quantiles` here if the behavior has changed in order to achieve parity between `OnboardingDataAssistant` and Auto-Initializing Expectations.

Reply:
✅ after our offline conversation